.github/workflows/go.yml

```diff
@@ -35,6 +35,9 @@ jobs:
       - name: Generate mocks
         run: make go_mockgen
 
+      - name: Run golangci-lint
+        run: make go_lint
```

Comment:
In CI, I would prefer us use an official [golangci-lint GitHub Action](https://github.com/marketplace/actions/run-golangci-lint). But I can hook it up in a subsequent PR if you would rather not deal with it. I was planning to add it along with our [custom linters from pocket repo](https://github.com/pokt-network/pocket/tree/main/build/linters). So thank you for taking the first step!